docs/kcl_mod.md

```diff
@@ -0,0 +1,70 @@
+# kcl.mod
```

Comment:
+ I don't have much problem with the structure definition. If you want to use it as a specification and user document, you can refer to some better documents for polishing, such as `cargo.toml`, and pay attention to compatibility and other issues. Change `kpm.lock` to `kcl.mod.lock`?
+ `kcl.mod` can use general structure definition methods such as protobuf on the basis of specifications to facilitate multilingual access.

Reply:
fixed.